autoload/neomake/makers/ft/python.vim

```diff
@@ -19,6 +19,10 @@ function! neomake#makers#ft#python#EnabledMakers()
         call add(makers, 'pylint')  " Last because it is the slowest
     endif
 
+    if neomake#utils#Exists('vulture')
```

Comment:
The exists check is done automatically.

Here it is used to e.g. only use `pylama` if it exists.
I think you want to just add it unconditionally.